packages/strapi-generate-new/lib/utils/parse-db-arguments.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const stopProcess = require('./stop-process');
+
+const dbArguments = [
+  'dbclient',
+  'dbhost',
+  'dbport',
+  'dbname',
+  'dbusername',
+  'dbpassword',
+];
+
+module.exports = function parseDatabaseArguments({ scope, args }) {
+  const argKeys = Object.keys(args);
+  const matchingDbArguments = dbArguments.filter(key => argKeys.includes(key));
+
+  if (matchingDbArguments.length === 0) return;
+
+  if (
+    matchingDbArguments.length !== dbArguments.length &&
+    args.dbclient !== 'sqlite'
+  ) {
+    return stopProcess(
+      `⛔️ Some database arguments are missing. Required arguments list: ${dbArguments}`
```

Comment:
`⛔️ Some database's arguments are missing. Required arguments list: ${dbArguments}`

Reply:
`⛔️ Some of the database arguments are missing. Required arguments list: ${dbArguments}`

database's is saying the database owns the arguments, typically not something stated in english as the database is an object and not a person (english confusing as objects can own something but in this context it's not quite "proper")